configure.ac

```diff
@@ -1,6 +1,17 @@
 AC_PREREQ([2.68])
 AC_INIT([rkt], [0.7.0+git], [https://github.com/coreos/rkt/issues])
 
+dnl if version ends with +git, append a short git-hash.
+AS_IF([test `expr match 'AC_PACKAGE_VERSION' '.*+git$'` -gt 0],
+      dnl version has +git suffix
+      [RKT_VERSION="AC_PACKAGE_VERSION`git rev-parse --short HEAD`"],
```

Comment:
I tried to compile this branch outside of git and I get:
```
$ git archive endocode/iaguis/optimization | tar -x -C /tmp/rkt-optimization
$ cd /tmp/rkt-optimization
$ ./autogen.sh ; ./configure
fatal: Not a git repository (or any parent up to mount point /tmp)
```

But the compilation then continues fine without the git id. Do we care about compilation outside of git?